### DISCUSSION TITLE
Rendering Bootstrap Grid

### DISCUSSION THREAD

**Discussion by sieglecj** [vikramlearning/blazorbootstrap#638] [Q&A]
Hi Everyone!  Noob question here.  I can't seem to get the Grid to render.  I followed the demo code for Server-side filtering, but the grid doesn't render when the page loads.  Also, the DataProvider async method doesn't get hit at all which makes me think I am missing something simple.  Do you have to create initialization code with the grid, or does the grid load data when the page loads?

**Comment by gvreddy04** [comment]
@sieglecj Thank you for trying **BlazorBootstrap**. Please share a sample code GitHub repo to reproduce the issue with minimal steps.
